### PR TITLE
Daval - experiment for reviewing purposes

### PR DESCRIPTION
**Pull request by DavalDavis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Elf Attack
-
-Now with slightly less ReadMe.
\ No newline at end of file
+This is another version that i will not merge immediately,
```
Comment (glitchbane) [inline README.md:2]: commeeeeeeennnnnnt

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+so we can see if Sheila can approve it.
```
Comment (WillSchutt) [inline README.md:3]: Is there somewhere we can "approve" the PR or would we just leave a comment to that effect?

Comment (DavalDavis) [inline README.md:3]: It doesn't look there's anywhere to approve, outside of leaving a comment to that effect.

**Comment by glitchbane** [comment]
- [ ] a task list

**Comment by glitchbane** [comment]
https://www.codereviewhub.com/?
it integrates with github, is free, and adds approval and tasking functionality.  I like it.

**Comment by glitchbane** [comment]
leave a comment!

File: README.md (hunk 2 above)
Comment (glitchbane) [inline README.md:3]: this is a comment.

**Pull request closed without merging**